.gitignore

```diff
@@ -59,3 +59,11 @@ typings/
 
 build/
 data/
+# OS generated files #
```

Comment:
Just something small, but a newline above would make it more consistent with the general file structure.

What is the idea behind `+# #########V#############` is this a divider thingy?